app-text/convmv/patches/convmv-2.01.patchset

```diff
@@ -0,0 +1,29 @@
+From f1f6e6e81e17d7ef0933c54c6f8f3b4d7e57cb7d Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sun, 21 May 2017 07:30:19 +0200
+Subject: import patch from v1.15
+
+
+diff --git a/Makefile b/Makefile
+index fdaf8bf..3df0c70 100644
+--- a/Makefile
++++ b/Makefile
+@@ -6,6 +6,7 @@ FIND=find
+ SED=sed
+
+ all: manpage
++	sed -i "s#/usr/bin/perl#`which perl`#" convmv
+
+ install: all
+ 	mkdir -p $(DESTDIR)$(MANDIR)/man1/
+@@ -22,6 +23,7 @@ clean:
+
+ test:
+ 	test -d suite || $(TAR) xf testsuite.tar
++	sed -i "s#/usr/bin/perl#`which perl`#" suite/parsable_tester.pl
```

Comment:
I'm not familiar how sed works, just imported the patch from the previous version :)
it would be something like:
sed -i "s#/usr/bin/perl#`s#/usr/bin/env perl`#" suite/parsable_tester.pl ?